lib/phlex/sgml/elements.rb

```diff
@@ -1,6 +1,39 @@
 # frozen_string_literal: true
 
 module Phlex::SGML::Elements
+	COMMA_SEPARATED_TOKENS = {
+		img: <<~RUBY,
+			if (tokens = attributes[:srcset])
+				attributes[:srcset] = __nested_tokens__(tokens, ", ")
+			end
+		RUBY
```

Comment:
The technique here is to generate specifically optimised code to check and pre-process the comma-separated-lists before caching takes place.